### PR TITLE
Updating Virtual Data Center doc with Ignite updates

### PR DESCRIPTION
**Pull request by tracsman**

### CODE REVIEW

File: docs/vdc/networking-virtual-datacenter.md [hunk 1]
```diff
@@ -24,72 +24,74 @@ The following cloud deployment diagram illustrates some examples of security gap
...
 
-Enterprise/organizations can require a demanding mix of services for different Line-of-Businesses (LOBs), and employees often have different roles when involved with different projects. A VDC requires good cooperation between different teams, each with specific role definitions, to get systems running with good governance. The matrix of responsibilities, access, and rights can be extremely complex. Identity management in VDC is implemented through [*Azure Active Directory* (AAD)][AAD] and Role-Based Access Control (RBAC).
+Enterprise/organizations can require a demanding mix of services for different Line-of-Businesses (LOBs), and employees often have different roles when involved with different projects. A vDC requires good cooperation between different teams, each with specific role definitions, to get systems running with good governance. The matrix of responsibilities, access, and rights can be extremely complex. Identity management in vDC is implemented through [*Azure Active Directory* (AAD)][AAD] and Role-Based Access Control (RBAC).
```
Comment (MikeWasson) [inline docs/vdc/networking-virtual-datacenter.md]: AAD should be "Azure AD"

File: docs/vdc/networking-virtual-datacenter.md [hunk 2]
```diff
@@ -24,72 +24,74 @@ The following cloud deployment diagram illustrates some examples of security gap
 
 [![0]][0]
 
-The Virtual Data Center (VDC) was born from this necessity for scaling to support enterprise workloads, and the need to deal with the problems introduced when supporting large-scale applications in the public cloud.
+The Virtual Data Center (vDC) was born from this necessity for scaling to support enterprise workloads, and the need to deal with the problems introduced when supporting large-scale applications in the public cloud.
```
Comment (MikeWasson) [inline docs/vdc/networking-virtual-datacenter.md]: VDC: We should stick with standard capitalization rules.

**Comment by MikeWasson** [comment]
@tracsman  please review the build report - there's an invalid link

File: docs/vdc/networking-virtual-datacenter.md [hunk 3]
```diff
@@ -78,15 +78,17 @@ The Azure fabric allocates infrastructure resources to tenant workloads and mana
 #### Connectivity to the cloud
 The VDC needs connectivity with external networks to offer services to customers, partners and/or internal users. This usually means connectivity not only to the Internet, but also to on-premises networks and data centers.
 
-Customers can build their security policies to control what and how specific VDC hosted services are accessible to/from the Internet using Network Virtual Appliances (with filtering and traffic inspection), and custom routing policies and network filtering (User Defined Routing and Network Security Groups).
+Customers can build their security policies to control what and how specific VDC hosted services are accessible to/from the Internet using [Azure Firewall][AzFW] or Network Virtual Appliances, and custom routing policies and network filtering ([User Defined Routing][UDR] and [Network Security Groups][NSG]). It's recommended that all Internet facing resources be additionally protected by [**Azure DDOS Protection Standard**][DDOS].
```
Comment (MikeWasson) [inline docs/vdc/networking-virtual-datacenter.md]: DDOS > DDoS

**Comment by tracsman** [comment]
Hi All, please review and publish or provide comments on whats needed, this needs to get published asap with the Ignite content. Thanks!

**Pull request merged**